src/test/data/XmlAddressBookStorageTest/invalidAndValidPersonAddressBook.xml

```diff
@@ -2,13 +2,15 @@
 <addressbook>
     <!-- Valid Person -->
     <persons>
+    	<nric>S1234567A</nric>
         <name>Hans Muster</name>
         <phone isPrivate="false">9482424</phone>
         <email isPrivate="false">hans@example.com</email>
         <address isPrivate="false">4th street</address>
```

Comment:
The person's attributes are not aligned here.

Reply:
My bad, I'll fix this in a future commit.